config.js

```diff
@@ -22,6 +22,10 @@ module.exports = {
 
   enabled: true,
 
+  // If true, information about query parameters and results will be
+  // attached to spans representating database operations.
+  enhancedReporting: false,
```

Comment:
Could we use a more descriptive name? Enhanced could mean anything. It also closes the door on us adding a really enhanced tracing mode later on :smile:.